test/representers/api/story_representer_test.rb

```diff
@@ -64,15 +64,6 @@
       json['_links']['profile']['href'].must_equal 'http://meta.prx.org/model/story'
     end
 
-    it 'has a profile for the default image' do
```

Comment:
profile is aways provided on the resource itself, but the link relation is supposed to be enough info about the linked resource. When we have had links to resources where there are many possible resources, and so the profile is not predictable or constant, we have specified a profile attribute with the other link attributes.

Now that the images will always be consistent about what profile they are for a given parent resource's link relation, no need to have profile in there - you'll see usually we don't have profiles for the vast majority of links (except self).